### PR TITLE
new metrics

### PR DESCRIPTION
**Pull request by dbernstein1**
[PMM-XXXX](https://jira.percona.com/browse/PMM-XXXX) (optional, if ticket reported)

- [ ] Links to other linked pull requests (optional).

---

- [ ] Tests passed.
- [ ] Fix conflicts with target branch.
- [ ] Update jira ticket description if needed.
- [ ] Attach screenshots/console output to confirm new behavior to jira ticket, if applicable.

Once all checks pass and the code is ready for review, please add `pmm-review-exporters` team as the reviewer. That would assign people from the review team automatically. Report any issues on our [Forum](https://forums.percona.com) or [Discord](https://per.co.na/discord).

### CODE REVIEW

File: exporter/v1_compatibility.go [hunk 1]
```diff
@@ -1200,9 +1222,118 @@ func chunksTotal(ctx context.Context, client *mongo.Client) (prometheus.Metric,
...
+
+	// 	metrics = append(metrics, metric)
+	// }
+	return metrics, nil
+}
+
+func chunksTotal(ctx context.Context, client *mongo.Client) (prometheus.Metric, error) {
+	n, err := client.Database("config").Collection("chunks").CountDocuments(ctx, bson.M{})
+	if err != nil {
+@ -1200,9 +1317,26 @@ func chunksTotal(ctx context.Context, client *mongo.Client) (prometheus.Metric,
```
Comment (BupycHuk) [inline exporter/v1_compatibility.go:1313]: this line looks pretty weird

**Comment by BupycHuk** [comment]
Hi, @dbernstein1 thanks for the contribution. Could you provide information on what this PR is about and what problem you are fixing here? Thanks

File: exporter/v1_compatibility.go (hunk 1 above)
Comment (dbernstein1) [reply exporter/v1_compatibility.go:1313]: I tried running make format but got these issues:

go mod tidy
go mod tidy: go.mod file indicates go 1.17, but maximum supported version is 1.16
make: *** [format] Error 1

**Comment by dbernstein1** [comment]
@BupycHuk 

1. added the following metrics: 
- collections with chunks currently being balanced among shards (mongodb_mongos_sharding_migrating)
- System balancer running on/off indictation (mongodb_mongos_sharding_balancer_running)
- chunk size in megabytes (mongodb_mongos_sharding_chunk_size_mb)

2. Included a label with the db/collection in mongodb_mongos_sharding_shard_chunks_total to see how chunks are balanced across shards in a given collection

3. Fixed a bug in balancerEnabled mongodb_mongos_sharding_balancer_enabled always returned 1 even when balanced was disabled

File: exporter/v1_compatibility.go (hunk 1 above)
Comment (BupycHuk) [reply exporter/v1_compatibility.go]: What version of golang do you have locally?

Comment (dbernstein1) [reply exporter/v1_compatibility.go]: 1.16.5

Comment (BupycHuk) [reply exporter/v1_compatibility.go]: Please update to at least 1.17.

**Comment by BupycHuk** [comment]
@dbernstein1 please sign the CLA

**Comment by dbernstein1** [comment]
@BupycHuk I already did but it is showing up that I didn’t 
Screen Shot 2022-10-21 at 3 53 01 PM

**Comment by dbernstein1** [comment]
Screen Shot 2022-10-21 at 3 55 25 PM

**Comment by artemgavrilov** [comment]
Hi @dbernstein1 your commits signed with `dabernst@cisco.com` email address, but your GH account has different address. Because of that CLA check is not passing.

**Comment by BupycHuk** [comment]
@dbernstein1 please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user)

**Comment by dbernstein1** [comment]
@BupycHuk I added dabernst@cisco.com to my github account. Tried signing the CLA again

Screenshot 2022-12-05 at 6 36 06 PM

**Comment by artemgavrilov** [comment]
@dbernstein1 @BupycHuk CLA problem has gone.

**Review by BupycHuk — CHANGES_REQUESTED** [review]
Hi @dbernstein1, Thank you for your contribution and fixing a problem with CLA.

v1_compatibilty collector is supposed to be a collector to keep compatibility with mongodb_exporter v1. For new metrics it's better to create a new collector.

**Comment by BupycHuk** [comment]
closed as outdated

**Pull request closed without merging**